ocs_ci/utility/ibmcloud.py

```diff
@@ -401,6 +402,24 @@ def get_data_volumes(self):
         logger.info(f"volume ids are : {vol_ids}")
         return vol_ids
 
+    def is_volume_attached(self, volume):
+        """
+        Check if volume is attached to node or not.
+
+        Args:
+            volume (str): The volume to check for to attached
+
+        Returns:
+            bool: 'True' if volume is attached
+            else 'False'
```

Comment:
```suggestion
            bool: 'True' if volume is attached otherwise 'False'
```

Make it one liner. Her you anyway are missing one indention to the right in the line where you start with `else`

Reply:
Updated the dock string to one line